.readthedocs.yml

```diff
@@ -0,0 +1,18 @@
+# This is the version for the readthedocs configuration. Version 2 ignores
+# web-based configuration and uses everything from this file.
+version: 2
+
+# Configure the python version and environment construction run before
+# docs are built.
+python:
+  version: 3.8
```

Comment:
Do we want this or 3.10?

Reply:
I doubt it matters as long as docs get built (and we DO support 3.8). This file was just copied from vivarium and I figure if it ain't broke, don't fix it

Reply:
Does it work with 3.10? If so, we should use it.

Reply:
will try.